c2corg_ui/static/js/documentediting.js

```diff
@@ -80,9 +81,9 @@ app.DocumentEditingController = function($scope, $element, $attrs,
 
   /**
    * @type {app.Authentication}
-   * @private
+   * @export
```

Comment:
These properties are indeed accessed by the child controllers and cannot be tagged as private. They do not really need to be exported. Should I simply remove the ``@private`` tag without adding an ``@export`` one?
If yes is the properties renaming appropriate (no trailing "_")?